platform/kube/inject/inject.go

```diff
@@ -209,7 +203,7 @@ func GetInitializerConfig(kube kubernetes.Interface, namespace, injectConfigName
 
 	// apply safe defaults if not specified
 	switch c.Policy {
-	case InjectionPolicyOff, InjectionPolicyDisabled, InjectionPolicyEnabled:
+	case InjectionPolicyDisabled, InjectionPolicyEnabled:
```

Comment:
line 191 "cannot configure both *include* namespaces and exclude namespaces" - include missing

Reply:
`namespaces` is intended to match the parameter name in the ConfigMap.